telegram/_messageentity.py

```diff
@@ -63,13 +63,26 @@ class MessageEntity(TelegramObject):
 
             .. versionadded:: 20.0
     Attributes:
-        type (:obj:`str`): Type of the entity.
+        type (:obj:`str`): Type of the entity. Can be :attr:`MENTION` (@username),
+            :attr:`HASHTAG`, :attr:`BOT_COMMAND`,
+            :attr:`URL`, :attr:`EMAIL`, :attr:`PHONE_NUMBER`, :attr:`BOLD` (bold text),
+            :attr:`ITALIC` (italic text), :attr:`STRIKETHROUGH`, :attr:`SPOILER` (spoiler message),
+            :attr:`CODE` (monowidth string), :attr:`PRE` (monowidth block), :attr:`TEXT_LINK` (for
+            clickable text URLs), :attr:`TEXT_MENTION` (for users without usernames),
+            :attr:`CUSTOM_EMOJI` (for inline custom emoji stickers).
+
+            .. versionadded:: 20.0
+                added inline custom emoji
```

Comment:
```suggestion
                Added inline custom emoji
```
can you do this change in the Args: section also?